nginx/conf.d/1_webapp.conf

```diff
@@ -0,0 +1,47 @@
+#Modified from: https://itnext.io/docker-rails-puma-nginx-postgres-999cd8866b18
+
+upstream safecastapi {
```

Comment:
I would appreciate it if you could leave a comment on this pull request to describe where you made changes and what changes you made.